Source/RapyutaSimulationPlugins/Private/Robots/Turtlebot3/TurtlebotBurger.cpp

```diff
@@ -14,13 +14,18 @@ ATurtlebotBurger::ATurtlebotBurger(const FObjectInitializer& ObjectInitializer)
     PrimaryActorTick.bCanEverTick = true;
 
     Init();
- 
+}
+
+void ATurtlebotBurger::InitializeMoveComponent() 
+{
+    DifferentialDriveComponent = NewObject<UDifferentialDriveComponent>(this, TEXT("DifferentialDriveComponent"));

```

Comment:
@Praphul Kallakuri For the crash that Yu might have just noted, though ATurtlebotBurger seems to be not using `RobotVehicleMoveComponent`, you could call `Super::InitializeMoveComponent()` here to initialize it, since in our parent class `ARobotVehicle` it is [hooked](https://github.com/rapyuta-robotics/RapyutaSimulationPlugins/blob/pk_refactor/Source/RapyutaSimulationPlugins/Private/Robots/RobotVehicle.cpp#L34) up to be initialized.
```suggestion
void ATurtlebotBurger::InitializeMoveComponent() 
{
    Super::InitializeMoveComponent();
    DifferentialDriveComponent = NewObject<UDifferentialDriveComponent>(this, TEXT("DifferentialDriveComponent"));

```

Reply:
@Praphul Kallakuri For the crash that Yu might have just noted, though ATurtlebotBurger seems to be not using `RobotVehicleMoveComponent`, you could call `Super::InitializeMoveComponent()` here to initialize it, since in our parent class `ARobotVehicle` it is currently [hooked](https://github.com/rapyuta-robotics/RapyutaSimulationPlugins/blob/pk_refactor/Source/RapyutaSimulationPlugins/Private/Robots/RobotVehicle.cpp#L34) up to be initialized.
```suggestion
void ATurtlebotBurger::InitializeMoveComponent() 
{
    Super::InitializeMoveComponent();
    DifferentialDriveComponent = NewObject<UDifferentialDriveComponent>(this, TEXT("DifferentialDriveComponent"));
```